.github/workflows/downstream.yml

```diff
@@ -24,24 +24,33 @@ jobs:
           python -m build .
           pip install dist/traitlets*.whl
           pip install --pre --upgrade traitlets[test] pytest pytest-cov
-          pip install ipython[test] git+https://github.com/jupyter/nbconvert.git#egg=nbconvert[test] \
+          pip install ipython[test] nbconvert[test] \
                       notebook[test] ipywidgets[test]
+          # Work around https://github.com/jupyter/nbconvert/issues/1754
+          pip install "bleach<5"
           pip freeze
 
       - name: Run tests IPython
         run: |
-          iptest
+          cd $HOME
```

Comment:
why `cd $HOME`?

Reply:
To avoid using our own pytest.ini as the settings